mkdocs/docs/HPC/FAQ.md

```diff
@@ -388,6 +388,17 @@ The `du` command returns the size of every file and subdirectory in the $VSC_HOM
 The `egrep` command will only let entries that match with the specified regular expression `[0-9]{3}M|[0-9]G` through, which corresponds with files that consume more than 100 MB.
 
 
+### How can I get more storage space?
+
+
+[By default](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#quota) you get 3 GB of storage space for your home directory and 25 GB in your personal directories on both the data (`$VSC_DATA`) and scratch (`$VSC_SCRATCH`) filesystems.
```

Comment:
Ideally we don't link to the full URL.
I should have raised this also for the other FAQ entries, to please also fix those in this PR.

Instead, you should point to the MarkDown file, and let `mkdocs` generate internal links that are correct, like this:

```suggestion
[By default](running_jobs_with_input_output_data.md#quota) you get 3 GB of storage space for your home directory and 25 GB in your personal directories on both the data (`$VSC_DATA`) and scratch (`$VSC_SCRATCH`) filesystems.
```

(see also existing FAQ entries where this is done)